protocol/x/clob/keeper/msg_server_cancel_orders.go

```diff
@@ -29,7 +29,7 @@ func (k msgServer) CancelOrder(
 			err,
 			types.ModuleName,
 			metrics.CancelOrder,
-			metrics.DeliverTx,
+			metrics.StatefulCancellationMsgHandler,
```

Comment:
why can't we keep DeliverTx here? The only time this function is called is in DeliverTx, so I'm wondering if this is a helpful distinction to make. Figured it would be nice to keep the Callback field to ABCI callbacks (or ones we've added like PCS)